dashboard/config/routes.rb

```diff
@@ -197,6 +197,7 @@
       get '/users/demigrate_from_multi_auth', to: 'registrations#demigrate_from_multi_auth'
       get '/users/to_destroy', to: 'registrations#users_to_destroy'
       get '/reset_session', to: 'sessions#reset'
+      get '/lockout', to: 'sessions#lockout'
```

Comment:
What if a student visits this URL and they have permission? I'm thinking we should redirect them to the /home page, but that work should be done in the global redirect task. I will note it there.